language-support/hs/bindings/src/DA/Ledger/Types.hs

```diff
@@ -187,17 +188,23 @@ data Value
     | VContract ContractId
     | VList [Value]
     | VInt Int
-    | VDecimal Text -- TODO: Maybe use Haskell Decimal type
-    | VString Text
-    | VTimestamp MicroSecondsSinceEpoch
+    | VDecimal Text
+    | VText Text
+    | VTime MicroSecondsSinceEpoch
     | VParty Party
     | VBool Bool
     | VUnit
     | VDate DaysSinceEpoch
     | VOpt (Maybe Value)
     | VMap (Map Text Value)
+    | VEnum Enum
     deriving (Eq,Ord,Show)
 
+data Enum = Enum
+    { eid   :: Maybe Identifier
+    , cons  :: ConstructorId
+    } deriving (Eq,Ord,Show)
```

Comment:
Could you make the various `data` / `newtype` definitions in this file more visually uniform? I'm partial to the style used in here in `data Enum`, but consistency is more important than any particular preference anyway.

Reply:
Yes! Totally Agree. I have recently migrated to the style used here for Enum. And have it on my to-do list to migrate this entire file to that format. But I didn't want to mix up a reformatting into this PR.

Reply:
:+1: